dataverse-persistence/src/main/resources/Bundle_pl.properties

```diff
@@ -2519,9 +2529,17 @@ facets.search.fieldtype.journalVolume_ss.label=Czasopismo Tom
 facets.search.fieldtype.journalIssue_ss.label=Czasopismo Numer
 facets.search.fieldtype.journalPubDate_ss.label=Czasopismo Data publikacji
 facets.search.fieldtype.journalArticleType_ss.label=Typ artyku\u0142u
+facets.search.fieldtype.authorName.authorName.label=Nazwisko i imi\u0119 lub nazwa autora
 #
-facets.search.fieldtype.fileTypeGroupFacet.label=Typ pliku
-facets.search.fieldtype.fileAccess.label=Dost\u0119p
+facets.search.fieldtype.access.label=Dost\u0119p
+facets.search.fieldtype.fileTypeGroupFacet.file_type.label=Typ pliku
+facets.search.fieldtype.fileTypeGroupFacet.tabular_data.label=Dane tabelaryczne
+facets.search.fieldtype.fileTypeGroupFacet.image.label=Obraz
+facets.search.fieldtype.fileTypeGroupFacet.unknown.label=Nieznany
+facets.search.fieldtype.fileTypeGroupFacet.document.label=Dokument
```

Comment:
We really need to find a different approach for this. But for now the all possible values of this are listed in values of file: MimeTypeFacets_en.properties
From there we can deduce that some values are missing, for example: Network Data